packages/hdwallet-tallyho/src/tallyho.ts

```diff
@@ -256,7 +256,7 @@ export class TallyHoHDWallet implements core.HDWallet, core.ETHWallet {
       // https://docs.metamask.io/guide/ethereum-provider.html#errors
       // Internal error, which in the case of wallet_switchEthereumChain call means the chain isn't currently added to the wallet
       if (error.code === -32603) {
-        // TODO: TallyHo currently supports a finite number of chains natively (Mainnet + Polygon/Arbitrum/Optimism under feature flag), with no capabilities to add new chains
+        // TODO: TallyHo currently supports a finite number of chains natively, with no capabilities to add new chains
```

Comment:
Support should be here soonish https://github.com/tahowallet/extension/blob/d5811da18605927dda92280092f4d41f3e432d58/background/services/internal-ethereum-provider/index.ts#L267